src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        return String.valueOf(Colors.values()[new Random().nextInt(Colors.values().length)]);
```

Comment:
Avoid creating a new instance of 'Random' every time the method is called. Instead, create a single 'Random' instance as a class-level variable and reuse it.